features/ubuntu_pro_fips.feature

```diff
@@ -527,6 +527,7 @@ Feature: Command behaviour when auto-attached in an ubuntu PRO fips image
         """
         1
         """
+        When I run `systemctl daemon-reload` with sudo
```

Comment:
I know this is unrelated, but those pro-fips tests are failing due to the extra check for the `esm-infra-security` when running apt-cache policy on the tested package. Maybe it is worth an extra commit just to address that and fix those tests as well

Reply:
Ah darn that wasn't happening when I was editing this. I'll see if I can fix it and tack on an extra commit here before merging